.github/workflows/docs.yml

```diff
@@ -91,16 +91,16 @@ jobs:
           path: moabb-ghio
           token: ${{ secrets.MOABB_GHIO }}
 
-      - name: Deploy on moabb.neurotechx.com
-        run: |
-          git config --global user.email "ci@neurotechx.com"
-          git config --global user.name "Github Actions"
-          cd ~/work/moabb/moabb/moabb-ghio
-          rm -Rf docs
-          cp -a ~/work/moabb/moabb/docs/build/html ./docs
-          git add -A
-          git commit -m "GH Actions update of docs ($GITHUB_RUN_ID - $GITHUB_RUN_NUMBER)"
-          git push origin master
+  deploy_neurotechx_subpages:
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/develop'}}
+    uses: peaceiris/actions-gh-pages@v3 # testing
```

Comment:
GH Action for this workflow indicates a problem on that line:
> Invalid workflow file: .github/workflows/docs.yml#L96
> invalid value workflow reference: references to workflows must be rooted in '.github/workflows'

Reply:
okay! trying to fix, thank for let me know!